### PR TITLE
Update Asset-Processing.json

### PR DESCRIPTION
**Pull request by amzn-mike**
## What does this PR do?

Added additional rule for matching CreateJobs failures

## How was this PR tested?

Just tested regex separately.

### CODE REVIEW

File: scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
         "find_all": true,
         "log_patterns": [
-          "^.*AssetProcessor: Failed.*\.\.\.$"
+          "^.*?AssetProcessor: Failed.*\.\.\.$",
+          "^.*?AssetProcessor: Createjobs Failed:.*$"
         ],
         "log_testcases": [
-          "AssetProcessor: Failed Shaders/Terrain/TerrainDetailClipmapGenerationPass.shader"
+          "AssetProcessor: Failed Shaders/Terrain/TerrainDetailClipmapGenerationPass.shader",
+          "AssetProcessor: Createjobs Failed: /data/workspace/o3de/Gems/LyShineExamples/Assets/UI/Fonts/LyShineExamples/NotoSerif/NotoSerif-Regular.xml."
```
Comment (amzn-changml) [inline scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json:15]: Is this something we should file bugs against?

**Review by amzn-changml — APPROVED** [review]
LGTM, aside from the non-blocking question

**Review by brianherrera — APPROVED** [review]

File: scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json (hunk 1 above)
Comment (shirangj) [reply scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json:15]: Good point! Creating a GHI is a better way to report the issue

**Review by shirangj — APPROVED** [review]

File: scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json (hunk 1 above)
Comment (amzn-mike) [reply scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json:15]: Do you mean filing a GHI for that specific error called out in the example or for this PR to fix the lack of reporting of CreateJobs failures?

Comment (amzn-changml) [reply scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json:15]: The specific error. Should font files in the example fail to process on a regular basis?

Comment (amzn-mike) [reply scripts/build/build_failure_rca/rca_patterns/Asset-Processing.json:15]: No its pretty rare to see that, I think the example I pulled from was due to some ScriptCanvas shutdown issue.  I'm not sure what this block in the JSON is used for, I was just trying to follow the existing format and put an example in there.

**Pull request merged**